Nodejs/Product/Nodejs/Guids.cs

```diff
@@ -65,7 +65,11 @@ internal static class Guids
         // Workspace guids
         public const string PackageJsonContextTypeString = "{78F43160-2968-4FCA-8829-7E30E9B610CF}";
         public readonly static Guid PackageJsonContextType = new Guid(PackageJsonContextTypeString);
-		
+
+        public const string WorkspaceExplorerDebugActionCmdSetString = "{5ea148a6-40af-4ff2-ab0f-60ed173c9f98}";
```

Comment:
>{ [](start = 72, length = 1)

We seem to be a little inconsistent about whether guids are surrounded by braces.

Reply:
Depends where we copied them from ;)